src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+import json
+from typing import Optional
+
+from ..security import Scope
+from . import APIDoc, APIRouter, CreatePermission, DeletePermission, Endpoint, \
+    EndpointDoc, ReadPermission, RESTController
+
+try:
+    from google.protobuf.json_format import MessageToJson
+
+    from ..services.nvmeof_client import NVMeoFClient
+except ImportError:
+    MessageToJson = None
+else:
```

Comment:
What's else here for? I suspect maybe that's the reason I don't see any nvmeof APIs listed in OpenAPI doc.

Reply:
Removing the `else:` fixed the issue. 
```suggestion
```

Reply:
the else is there to make sure this code is run only when the import is successful otherwise, this would fail and cause issues to load dashboard module.

And the reason you are not seeing this in docs might be because the openapi spec is not generated, which i forgot to do. So I'll just quickly generate it and update the PR. thanks

Reply:
I've added the deps and pushed again. But I am not sure if the checks will pass this time since I added the `grpcio` and `grpcio-tools` in the requirments file and these are not available in centos8. let's see how it goes